src/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,316 @@
+                primary={avgBlockTime.value ? dayjs.duration(avgBlockTime.value).format('m[m] s[s]') : '-'}
+                secondary="of all shards"
+              />
+            </Card>
+          </StatisticsContainer>
+        </StatisticsSection>
+        <LatestsBlocks>
+          <Heading>Latest Blocks</Heading>
+          <Content>
+            <BlockListTable main isLoading={blockPageLoading} minHeight={498}>
+              <TableHeader
+                headerTitles={['Height', 'Timestamp', 'Txn', 'Chain index']}
+                columnWidths={['20%', '30%', '20%', '25%']}
+              />
+              <TableBody tdStyles={TableBodyCustomStyles}>
+                {blockList &&
+                  blockList.blocks?.map((b) => (
+                    <TableRow
+                      key={b.hash}
+                      onClick={() => {
+                        history.push(`blocks/${b.hash}`)
+                      }}
+                    >
+                      <td>
+                        <BlockHeight>{b.height.toString()}</BlockHeight>
+                      </td>
+                      <td>
+                        <Timestamp timeInMs={b.timestamp} />
+                      </td>
+                      <td>{b.txNumber}</td>
+                      <td>
+                        {b.chainFrom} → {b.chainTo}
+                      </td>
+                    </TableRow>
+                  ))}
+              </TableBody>
+            </BlockListTable>
+          </Content>
+          <PageSwitch totalNumberOfElements={blockList?.total} />
+        </LatestsBlocks>
+      </MainContent>
+      <Waves />
+      <AnimatePresence initial={false}>
+        {detailsCardOpen && (
+          <FullScreenCard
+            layoutId={`expandableCard-${detailsCardOpen}`}
+            label={fullScreenCardLabels[detailsCardOpen]}
+            key={detailsCardOpen}
+            onClose={() => setDetailsCardOpen(undefined)}
+          >
+            <LineAreaChart
+              series={vectorData[detailsCardOpen].value.series}
+              categories={vectorData[detailsCardOpen].value.categories}
+              colors={chartColors[detailsCardOpen]}
+              xAxisType="datetime"
+              yAxisType="tx"
+            />
+          </FullScreenCard>
+        )}
+      </AnimatePresence>
+    </StyledSection>
+  )
+}
+
+const fullScreenCardLabels: Record<VectorStatisticsKey, string> = {
+  txPerDay: 'Transactions per day',
+  hashratePerDay: 'Hashrate per day'
+}
+
+const chartColors: Record<VectorStatisticsKey, [string, string]> = {
+  txPerDay: ['#16cbf4', '#8a46ff'],
+  hashratePerDay: ['#b116f4', '#ff904c']
+}
+
+const StyledSection = styled(Section)`
+  margin-top: -30px;
+`
+
+const MainContent = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 40px;
+  align-self: center;
+
+  @media ${deviceBreakPoints.mobile} {
+    flex-direction: column;
+    gap: 30px;
+  }
+`
+
+const Heading = styled.h2``
```

Comment:
Isn't an `<h2>` semantic enough for a heading? :)

Reply:
To be honest, those are simply the remanent objects of previously styled components that have been simplified. 
I kept them there just because... I don't really like how the "native" html elements appear compared to the styled components 🙃 

<img width="293" alt="image" src="https://user-images.githubusercontent.com/6715943/171123286-0b6bdf35-9e85-4ab4-9979-0afcdfa8e680.png">

I know, it's not a very good reason but hey. I'll remove them.